SHTSensor.h

```diff
@@ -157,7 +166,7 @@ class SHTSensorDriver
    * Set the sensor accuracy.
    * Returns false if the sensor does not support changing the accuracy
    */
-  virtual bool setAccuracy(SHTSensor::SHTAccuracy newAccuracy) {
+  virtual bool setAccuracy(__attribute__((unused)) SHTSensor::SHTAccuracy newAccuracy) {
```

Comment:
I'm not sure how portable `__attribute__((unused))` is and if it works with all Arduino compilers.

Reply:
should I just update this to ```SHTSensor::SHTAccuracy /* newAccuracy */```, which should fix the warning as well?

Reply:
> I figured Arduino is mainly g++, but valid concern.

I'm not sure what compiler the ESP32 based Arduino's use, but AFAIK it's not gcc/g++.

Alternatively you can just do:
```cpp
   virtual bool setAccuracy(__attribute__((unused)) SHTSensor::SHTAccuracy newAccuracy) {
       (void)newAccuracy; // silence warning about unused argument
       return false;
   }
```